src/services/itemApi.jsx

```diff
@@ -0,0 +1,17 @@
+export async function getProducts(params = {}) {
```

Comment:
P2:
이렇게 param을 받으면 어떤 값이든 넘어올 수 있게 됩니다.
가능하면 받을 수 있는 값들을 제한해, 사용자가 어떤 값을 넘길 수 있는지 알 수 있게 해주세요.

```suggestion
async function getData(params = {}) { ... }
export async function getProducts({page, pageSize, orderBy, keyword }) { /** 해당 함수 내부에서 getData 함수 호출 *//}
```